samples/compute/v2/servers/list_volume_attachments.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use OpenStack\BlockStorage\v2\Models\VolumeAttachment;
+use OpenStack\Networking\v2\Extensions\SecurityGroups\Models\SecurityGroup;
+use OpenStack\Networking\v2\Extensions\SecurityGroups\Models\SecurityGroupRule;
+
+require 'vendor/autoload.php';
+
+$openstack = new OpenStack\OpenStack([
+    'authUrl' => '{authUrl}',
+    'region'  => '{region}',
+    'user'    => [
+        'id'       => '{userId}',
+        'password' => '{password}'
+    ],
+    'scope'   => ['project' => ['id' => '{projectId}']]
+]);
+
+$compute = $openstack->computeV2(['region' => '{region}']);
+
+$server = $compute->getServer(['id' => 'uuid']);
+
+foreach ($server->listVolumeAttachments() as $volumeAttachment) {
+    /**@var VolumeAttachment $volumeAttachment*/
+    print_r($volumeAttachment);
```

Comment:
so far, we've avoided doing any kind of variable outputting in loop examples because i don't think it really demonstrates anything. what do you think?